lib/ibge/reader/helpers.rb

```diff
@@ -0,0 +1,63 @@
+module Ibge
+  module Reader
+    module Helpers
+      SOURCES = {
+        dtb: 'DTB_2014_subdistrito.xls',
+        sidra: 'sidra.xls'
+      }
+
+      attr_reader :defined_columns, :selected_source
+
+      def columns(columns = {})
+        @defined_columns ||= {}
+        @selected_source ||= :dtb
+
+        distinct_sources = []
+
+        columns.each do |k, v|
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.